src/components/styled/StyledNew.js

```diff
@@ -0,0 +1,62 @@
+import styled from 'styled-components';
+
+export const StyledHeadbanner = styled.div`
+	padding: 22px 20px;
+	background-color: #009150;
+	font-size: x-large;
+	text-align: center;
+`;
+
+export const StyledNavUl = styled.div`
```

Comment:
Why a  `div` and not a `list` element? Your name would suggest that the styling is based on a list element.